tools/releaseBuild/vstsbuild.ps1

```diff
@@ -0,0 +1,59 @@
+param(
+    [Parameter(ParameterSetName='Build')]
+    [ValidatePattern("^v\d+\.\d+\.\d+(-\w+\.\d+)?$")]
+    [string]$ReleaseTag,
+
+    [ValidateSet('win7-x86','win7-x64','ubuntu.14.04','ubuntu.16.04','centos.7')]
+    [String]
+    $Name
+)
+$ErrorActionPreference = 'Stop'
+
+
+$psReleaseBranch = 'master'
+$psReleaseFork = 'PowerShell'
+$location = Join-Path -Path $PSScriptRoot -ChildPath 'PSRelease'
+if(Test-Path $location)
+{
+    Remove-Item -Path $location -Recurse -Force
+}
+
+$gitBinFullPath = (Get-Command -Name git).Source
+if (-not $gitBinFullPath)
+{
+    throw "Git is required to proceed. Install from 'https://git-scm.com/download/win'"
+}
+
+Write-Verbose "cloning -b $psReleaseBranch --quiet https://github.com/$psReleaseFork/PSRelease.git" -verbose
+& $gitBinFullPath clone -b $psReleaseBranch --quiet https://github.com/$psReleaseFork/PSRelease.git $location
```

Comment:
Why not move those foundation modules to powershell repo? Wouldn't it be clear/easier to find everything in one place?

Reply:
I need to deliver this to other teams.  PowerShell shouldn't be the one big monolithic project that owns very module.

Reply:
OK, so we want those modules to be reused by other teams. That makes sense.